guides/wayback-integration-guide.rst

```diff
@@ -30,15 +30,15 @@ Following this introduction, the Wayback Integration Guide includes the followin
 -   **More information** - Provides some links for more information.
 
 
-Wayback vs OpenWayback
-======================
+Wayback vs OpenWayback vs PyWayback
+===================================
 
-Two options exist currently for Wayback: `Wayback <http://archive-access.sourceforge.net/projects/wayback/>`_ from the
-Internet Archive or `OpenWayback <http://netpreserve.org/openwayback>`_ from IIPC. Web Curator Tool was originally
-developed and tested with Wayback. However OpenWayback is more actively developed at the moment.
+There are a number of options for Wayback, including `Wayback <http://archive-access.sourceforge.net/projects/wayback/>`_ from the
+Internet Archive, `OpenWayback <http://netpreserve.org/openwayback>`_ from IIPC and `PyWB <https://pywb.readthedocs.io/en/latest/>`_ from Webrecorder. Web Curator Tool was originally
```

Comment:
Should be 'Rhizome' instead of webrecorder

Reply:
Thanks for that.  I've made that change.